rust/utils/abigen/src/lib.rs

```diff
@@ -1,12 +1,20 @@
-use fuels_code_gen::ProgramType;
 use std::collections::BTreeSet;
 use std::ffi::OsStr;
 use std::fs::{self, File};
 use std::io::Write;
 use std::path::Path;
 
+// use fuels_code_gen::ProgramType;
```

Comment:
Should remove commented out code

Reply:
The sway version of this pr can be already not latest, you can reuse this code to update hyperlane to beta-4